.github/workflows/vulncheck.yml

```
@@ -21,7 +21,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.22.3
+          go-version: 1.22.4
```

```suggestion
          go-version: 1.22.x
```

Is there a point to manually upgrade this?

This isn't linked to the build system anyway. What does that use?

---

.x doesn't work @klauspost thats why its kept with actual version.

---

Its @aead suggestion that we must always run with actual version for vulncheck this basically tells us that we must make new releases as there are vulnchecks failing.

The version here sticks to the version we use to make releases.

---

https://github.com/minio/kes/blob/fe54489812091915144bed6c3a793312fce31e09/.github/workflows/go.yml#L73

---

@harshavardhana I don't suppose this is extracted for the build. 

You could run it on the *build* version when doing a release. This tells you nothing, except being annoying every time a minor release is made.

---

We can remove this once we add `vulncheck` to release process.

---

// cc @donatello